src/domain/event/eventCard/LargeEventCard.tsx

```diff
@@ -2,7 +2,7 @@ import classNames from 'classnames';
 import { Button } from 'hds-react';
```

Comment:
This is not related to this PR. But there are quite much same logic on  EventCard and LargeEventCard. The layout and buy button are pretty much only differences. I know this was my decision but it could be good idea to have all the logic in single component. Could be little bit easier to maintain.